test/library/standard/Sort/errors/errors-comparator.compareargs.good

```diff
@@ -1,2 +1,2 @@
 errors-comparator.chpl:9: In function 'main':
-errors-comparator.chpl:17: error: The comparator compareargs requires a 'key(a)', 'compare(a, b)', or 'keyPart(a, i)' method for element type int(64)
+errors-comparator.chpl:17: error: The comparator compareargs must implement either 'keyComparator', 'keyPartComparator', or 'relativeComparator' for int(64)
```

Comment:
I think this error message would be confusing.  Shouldn't it say something along the lines of "the comparator compareargs implements keyPartComparator but doesn't correctly provide a keyPart method"?

Reply:
Done in https://github.com/chapel-lang/chapel/pull/25852/commits/23295bd176186b2503c3b8e9ce18194ae937cdd4